nativeauthenticator/handlers.py

```diff
@@ -133,7 +136,7 @@ async def post(self):
         if assume_human:
             user_info = {
                 "username": self.get_body_argument("username", strip=False),
-                "pw": self.get_body_argument("pw", strip=False),
+                "signup_password": self.get_body_argument("pw", strip=False),
```

Comment:
Will this work to rename? I think this us later passed to create_user, and then used to set values as named like this on a UserInfo object, which maps to how the database table stores information - which shouldn't be changed.

Reply:
I think perhaps what you want to do is this, which would read the information from the form named `signup_password` as it is now named in the HTML I think.

```suggestion
                "pw": self.get_body_argument("signup_password", strip=False),
```

Reply:
Good catch. I have incorporated this change.